setup.py

```diff
@@ -47,10 +47,10 @@ def read(*rnames):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8-pre',
```

Comment:
I do not see Programming Language :: Python :: 3.8-pre on https://pypi.org/classifiers/

Reply:
Just updated it. Thanks for the review.